test/integration/simple-deployment/00-assert.yaml

```diff
@@ -1,3 +1,16 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
The problem with this test is that you are requesting the logs before the pod is actually prepared (in Running state), so you have 2 options to fix this:

1. Keep it in 00-assert.yaml file but check for Pod running like here https://github.com/keptn/lifecycle-toolkit/pull/1834/files#diff-d843194edcb9f505394bbf174edb253c18c96be819f1b5bf2b1079de7247227bR8
2. move it to 01-assert.yaml (I would prefer this option)